app/main/routes/federated.py

```diff
@@ -418,17 +251,16 @@ def auth():
     _auth_token = data["auth_token"]
     model_name = data.get("model_name", None)
 
+    server, _ = process_manager.get_configs(name=model_name)
+
     """stub DB vars"""
-    JWT_VERIFY_API = 1  # maybe processes._processes.last()["server_config"].get("JWT_VERIFY_API", None)
-    RSA = False
-    # maybe processes._processes.last(name=model_name)["server_config"].get("JWT_with_RSA", True)
+    JWT_VERIFY_API = server.config.get("JWT_VERIFY_API", None)
+    RSA = server.config.get("JWT_with_RSA", None)
+
     if RSA:
-        pub_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDAswWWr/kU9Z5kj7KIQEs54B9x1MaEhEp4WDZPJ+PGONfg2tD4BKuGtDl345f4zgx7EPZL7EZRApLq6HxcznVbLleIbyqKkzvR88zHLBaxQ9GBRx+0kH8VqZspmMI/6fDBVm/SDtG1GOAYPwX1zK3DZZFMkkA2v8oGZ3U791jd9gy7S5CxewJrFMcFMStj9x8x3tW07OAdC7/HZpa5zKE2rWN01tytxbsl9/coMNBAfWIWEflhZgRz2+Onp2uDaXez7RNTe4m0+tQlx2FD0Pb7rFvlKwsgziKBReO8wwCQXWqcAPLsIXCOfUZXlBNpvPvp9I4HPEffaHyR1FC2eRoj4hzUibEu0+OQNj7QM5P9KsMV9k4wxURFxsd78rlFF8cnbKwIMf5nB8/FbqL/IyJOggxtntHr1Gum44QnG794GtSQHZNlWKKak2z/u2O++flxfZ9dBBAYWjJYM5kIT+X9NVYbWWryBqupHYipwP8f3vovKWVacOMMm3S0z76O5IDiIp5Gjnsifbnz57FWQok0HrSv8l3QMRPCxi3SjIFyI2ZusFC/4VLy9zZXQe07qI6l7s91UN6W8VW1YUFQ7nLGffkpAd/bLZSOueYQrf5tslQjZf3Jon5C/MkTJ7PGyOUmoAYya2kyKi4izMg/ODRIloVbWjU6tEPWyhzK8VMsXw== root@388da63cf68e"
-        # @TODO: remove when hooked up to DB
-        # maybe processes._processes.last(name=model_name)["server_config"].get("pub_key", None)
+        pub_key = server.config.get("pub_key", None)
     else:
-        SECRET = "very long a$$ very secret key phrase"  #  TODO:@PRTFW remove after hookup to DB  # maybe processes._processes.last(name=model_name)["server_config"].get("JWT_SECRET", "very long a$$ very secret key phrase")
-
+        SECRET = server.config.get("JWT_SECRET", "very long a$$ very secret key phrase")
     """end stub DB vars"""
```

Comment:
@Prtfw Could you verify/test if I fixed it properly?

Reply:
Yes. I will, feel free to merge and I can test that and submit issues for stuff that doesn't? if you prefer i test before merge, I can do that too : )

Reply:
@IonesioJunior yes, can def do that, do you want to merge first and I can submit issues for the parts not working or do you want me to test first?

Reply:
It's up to you! :)

Reply:
If you are done / happy with the refactors, lets maybe merge this `big boy` first... I don't want to hold up the PR train : )
And we can start testing JWT as a separate issue. 
(I am starting to test but don't think the PR has to wait)